hddm/models/hddm_regression.py

```diff
@@ -22,7 +22,7 @@ def generate_wfpt_reg_stochastic_class(wiener_params=None, sampling_method='cdf'
                          'w_outlier': 0.1}
     wp = wiener_params
 
-    def wiener_multi_like(value, v, sv, a, z, sz, t, st, reg_outcomes, p_outlier=0):
+    def wiener_multi_like(value, v, sv, a, z, sz, t, st, alpha, reg_outcomes, p_outlier=0):
```

Comment:
I don't think you need to add alpha to anything outside of your new class.

Reply:
I've tried adding alpha directly in the HDDMrlRegressor class (see below for what I've tried), but then it does not allow running regressors onto alpha. 

from HDDMrlRegressor.py under class HDDMrlRegressor:

def _create_wfpt_knode(self, knodes):
        wfpt_parents = self._create_wfpt_parents_dict(knodes)
        wfpt_parents['alpha'] = knodes['alpha_bottom']
        return Knode(self.wfpt_reg_class, 'wfpt', observed=True,
                     col_name=['split_by', 'feedback', 'response', 'rt', 'q_init'],
                     reg_outcomes=self.reg_outcomes, **wfpt_parents)

    def _create_stochastic_knodes(self, include):
        # Create all stochastic knodes except for the ones that we want to replace
        # with regressors.
        knodes = super(HDDMrlRegressor, self)._create_stochastic_knodes(include.difference(self.reg_outcomes))
        knodes.update(self._create_family_normal(
                    'alpha', value=0, g_mu=0.2, g_tau=3**-2, std_lower=1e-10, std_upper=10, std_value=.1))

Reply:
Did you overload `__init__()`?

Reply:
I've moved the wfpt_parents['alpha'] to inside the HDDMrlRegressor class and removed alpha from all other likelihoods. And the HDDMrlRegressor works as before, i.e. allows regressing against alpha.